script/autoapi/nettest.hpp.j2

```diff
@@ -0,0 +1,463 @@
+// Part of Measurement Kit <https://measurement-kit.github.io/>.
+// Measurement Kit is free software under the BSD license. See AUTHORS
+// and LICENSE for more information on the copying conditions.
+#ifndef MEASUREMENT_KIT_NETTEST_HPP
+#define MEASUREMENT_KIT_NETTEST_HPP
```

Comment:
Is this file readable? (I have already commented on specific choices in `nettest.hpp`.)